style1.css

```diff
@@ -0,0 +1,418 @@
+
+a {
+  text-decoration:none;
+  color: #808080;
+}
+
+body {
+  margin:0 auto;
+  max-width: 1020px;
+
+}
+
+li {
+    list-style-type: none;
+   }
+
+.upper {
+  display: flex;
+}
+
+
+.black {
+  display: flex;
+  width:25px;
+  height:70px;
+  background-color: #808080;
+}
+
+.gray {
+  display: flex;
+  width:50px;
+  height:70px;
+  background-color: #C0C0C0;
+}
+
+.orange {
+  display: flex;
+  width:25px;
+  height:70px;
+  background-color: #FF7F50;
+}
+
+.header_text {
+  justify-content: center;
+  align-items: center;
+  font-family:Bradley Hand;
```

Comment:
if font has two words in the title it would be good to use the font name in quotes